manifests/jumpbox.yml

```diff
@@ -0,0 +1,37 @@
+name: jumpbox
+
+stemcells:
+  - alias:   default
+    os:      ubuntu-xenial
+    version: latest
+
+releases:
+  - name:    containers
+    version: latest
+
+update:
+  canaries:          1
+  max_in_flight:     1
+  serial:            true
+  canary_watch_time: 1000-120000
+  update_watch_time: 1000-120000
+
+instance_groups:
+  - name: docker
+    instances: 1 
+    azs: [z1]
+    vm_type: default
+    stemcell: default
+    networks:
+      - name: default
+    jobs:
+      - name: docker
+        release: containers
+
+      - name: jumpbox
+        release: containers
+        properties:
+          users:
+            - username: jhunt
+              key:      ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKF3JPSfD1e92ryLKxYlSlXtDmbUUe0EOalBAJvthCmK jhunt@hydra.local
```

Comment:
Should we generalize the manifest?

Reply:
meh.  I thought about it, but then it (a) wouldn't really be deployable and (b) I wouldn't have unfettered access to people's jumpboxen.